modular_skyrat/modules/GAGS/json_configs/pants_shorts_skirts_dresses/shortdress.json

```diff
@@ -0,0 +1,16 @@
+{
+	"shortdress": [
```

Comment:
Just making sure, any reason we can't have those icon_states follow `snake_case` conventions? I figured I'd ask, since you're there, as it would probably be a very simple change to make now rather than later.

Reply:
original one didnt, it slipped my mind to change it when I changed the original to greyscale; it'll be fixed